src/mk.sh

```diff
@@ -75,6 +75,26 @@ function get_kernel_release
   cmd_manager "$flag" "$cmd"
 }
 
+# When kw deploy a new kernel it creates temporary files to be used for moving
+# to the target machine. There is no need to keep those files in the user
+# machine, for this reason, this function is in charge of cleanup the temporary
+# files at the end.
+#
+# @flag How to display a command, the default value is
+#   "SILENT". For more options see `src/kwlib.sh` function `cmd_manager`
+function cleanup_after_deploy
```

Comment:
If the user aborts execution (with `ctrl-c`, for example), the cache doesn't get removed and the next execution might get dirty files. I think we could add two changes to help avoiding this:

1. Use bash's `trap` command, to trap termination signals and cleanup cache before exiting.
2. Always call `cleanup_after_deploy` **before** deploy as well (and maybe rename the func?)

Reply:
Uouuu I love the idea of handling trap! However, I believe that add traps should be a separate patch since we have to handle other cases. I'm going to create an issue for support to trap.